srcpkgs/ImageMagick/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'ImageMagick'
 pkgname=ImageMagick
 _majorver=7.0.11
-_patchver=6
+_patchver=12
```

Comment:
Could you change to `version=7.0.11.12` `_majorver=${version%.*}` `_patchver=${version##*.}` to eliminate substitions in version?

Reply:
ok, done.